tox.ini

```diff
@@ -5,11 +5,11 @@ envlist =
     mypy
 
     # Twisted 15.5 is the first version to support Python 3 (3.3+)
-    coverage-py{27,py,34,35,36}-tw{155,166,171,current,trunk}
+    coverage-py{27,34,35,36,py2,py3}-tw{155,166,171,current,trunk}
```

Comment:
If 15.5.0 isn't tested in Travis, it should be removed from `tox.ini`, since I'd rather not run a test locally that doesn't affect my code's mergability.